test/conformance/api/v1/service_test.go

```diff
@@ -36,6 +36,49 @@ import (
 	rtesting "knative.dev/serving/pkg/testing/v1"
 )
 
+// TestServiceCreateListAndDelete tests Creation, Listing and Deletion for Service resources.
+//   This test doesn't validate the Data Plane, it is just to check the Control Plane resources and their APIs
+func TestServiceCreateListAndDelete(t *testing.T) {
+	t.Parallel()
+	clients := test.Setup(t)
+
+	names := test.ResourceNames{
+		Service: test.ObjectNameForTest(t),
+		Image:   test.PizzaPlanet1,
+	}
+
+	// Clean up on test failure or interrupt
+	test.EnsureTearDown(t, clients, &names)
+
+	// Setup initial Service
+	_, err := v1test.CreateServiceReady(t, clients, &names)
+	if err != nil {
+		t.Fatalf("Failed to create initial Service %v: %v", names.Service, err)
+	}
+
+	// Validate State after Creation
+
+	if err = validateControlPlane(t, clients, names, "1"); err != nil {
+		t.Error(err)
+	}
+
+	list, err := v1test.GetServices(clients)
```

Comment:
You probably want to handle the error explicitly.

Reply:
done